### ISSUE TITLE
Composer init is failing without any reason.

### ISSUE THREAD

**Issue by brunoczim** [composer/composer#7056]
Output of `composer diagnose`:

```
Checking platform settings: OK
Checking git settings: OK
Checking http connectivity to packagist: OK
Checking https connectivity to packagist: OK
Checking github.com rate limit: OK
Checking disk free space: OK
Checking pubkeys: 
Tags Public Key Fingerprint: 57815BA2 7E54DC31 7ECC7CC5 573090D0  87719BA6 8F3BB723 4E5D42D0 84A14642
Dev Public Key Fingerprint: 4AC45767 E5EC2265 2F0C1167 CBBB8A2B  0C708369 153E328C AD90147D AFE50952
OK
Checking composer version: OK
Composer version: 1.6.2
PHP version: 7.0.27-0+deb9u1
PHP binary path: /usr/bin/php7.0

```

When I run this command:

```
bruno@debian:~/Documents/php/tester/testpkg$ composer init
```

I get the following output:

```

                                            
  Welcome to the Composer config generator  
                                            


This command will guide you through creating your composer.json config.

Package name (<vendor>/<name>) [bruno/testpkg]: 
                                                          
  [Symfony\Component\Console\Exception\RuntimeException]  
  Aborted                                                 
                                                          

init [--name NAME] [--description DESCRIPTION] [--author AUTHOR] [--type [TYPE]] [--homepage HOMEPAGE] [--require REQUIRE] [--require-dev REQUIRE-DEV] [-s|--stability STABILITY] [-l|--license LICENSE] [--repository REPOSITORY]
```

And I expected this to happen:
```
Package name (<vendor>/<name>) [bruno/testpkg]: <me entering some input>
... etc
```
The directory is empty.
`ls -mod` outputs:  `drwxr-xr-x 2 bruno 4096 Jan 29 12:27 .`

**Comment by brunoczim** [comment]
It seems like creating a new session in the terminal fixes the problem, but I have no idea why.

**Comment by glensc** [comment]
did it abort immediately when it was supposed to wait for terminal input? https://asciinema.org/ recording could show timing better.

**Comment by Seldaek** [comment]
@brunoczim can you reliably reproduce it or was it a one off and restarting the terminal fixed it for good?

**Comment by efimov90** [comment]
Same error 1.9.1 (stable channel), php 7.4.0 ( NTS Visual C++ 2017 x64 )

**Comment by Steve51D** [comment]
I get the same error on the same versions. 
Fresh install of PHP and composer. (Windows 10)
When I run `composer init` I get:

`
  Welcome to the Composer config generator



This command will guide you through creating your composer.json config.

Package name (<vendor>/<name>) [stephenadmin/pipeline-php]:

  [Symfony\Component\Console\Exception\RuntimeException]
  Aborted`

Restarting the terminal does not help. I also tried reinstalling and running as admin but no luck. Not sure what else to try.

**Comment by stof** [comment]
PHP 7.4.0 for windows has a bug breaking the QuestionHelper (it does not block until you write the input when reading from STDIN). Until PHP releases the fix (a PR is already in progress for it), you should switch back to PHP 7.3.

**Comment by stof** [comment]
That issue is a duplicate of https://github.com/composer/composer/issues/8457

**Comment by Nimash68** [comment]
Your package name should not contain uppercase characters.

**Comment by Chaufilipe** [comment]
Hello guys, i had the same bug, but I after all I realized that it happens in the last version of php. Example: I am now using PHP 7.4 but when I switched to PHP 7.3.12 the bug was solved! So I could dare saying that for that specific bug the first thing to check is the version of PHP you're using whether is it stable of not..

**Comment by stof** [comment]
@Chaufilipe this bug was fixed in 7.4.1 (a long time ago). So I would say that the thing to check is whether you run an uptodate version of PHP (PHP 7.4 is stable since one year)

**Comment by CheaVarin** [comment]
.
